recipe/recipe.yaml

```diff
@@ -1,27 +1,22 @@
 context:
   version: "1.29.0"
-  python_min: '3.9'
+  python_min: "3.9"
+  build: 1
```

Comment:
will this work with autotick bot @ytausch?

Reply:
I would avoid an additional variable if it's avoidable whenever possible

Reply:
@ytausch @pavelzw I did run into the problem now that I want to be able to use the build number when writing the build string: 

```yaml
string: h${{ hash }}_${{ polars_variant | replace('-', '_') }}_${{ build_number }}
```

The only way I can figure out to accomplish this without having to define the build number in two different places is by having `build_number` be a variable in `context`, and then also using that variable in the `context` section. I also couldn't find any [other feedstocks](https://github.com/search?q=org%3Aconda-forge+path%3Arecipe.yaml+%22string%3A%22&type=code) that do a better job.

Reply:
yes i think this is the way to do it at the moment